service/src/main/java/uk/gov/hmcts/reform/fpl/model/SupportingEvidenceBundle.java

```diff
@@ -17,7 +17,7 @@
 public class SupportingEvidenceBundle {
     private final String name;
     private final String notes;
-    @PastOrPresent(message = "Date of time received cannot be in the future")
+    @DateNotInFuture()
```

Comment:
should it not have specific message, default one, I believe is not really precise.

Reply:
alternativelly

@PastOrPresent
@JsonIgnore
LocalDate getDateReceived(){
return dateTimeReceived.toLocalDate();
} (edited)